test/gtest/ucs/test_event_set.cc

```diff
@@ -0,0 +1,140 @@
+/**
+* Copyright (C) Hiroyuki Sato. 2019. ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#include <common/test.h>
+extern "C" {
+#include <ucs/sys/event_set.h>
+#include <sys/epoll.h>
```

Comment:
this test includes epoll.h, how will it work on MacOS?

Reply:
see previous comment - the API should be platform-independent

Reply:
Removed `epoll.h`. new test is platform independent.